README.md

```diff
@@ -36,6 +36,7 @@ Maximum length of 60 characters.
 ### `content`
 
 **Required** The content of the message. Maximum message size of 10000 bytes.
+Accept [GitHub Flavored Markdown](https://zulip.com/help/format-your-message-using-markdown) syntax.
```

Comment:
Maybe 

"Format your message using [Zulip Markdown](https://zulip.com/help/format-your-message-using-markdown)."

I think that's clearer than referencing GFM, which Zulip does not exactly replicate and does not aim to.

Reply:
```suggestion
Format your message using [Zulip Markdown](https://zulip.com/help/format-your-message-using-markdown).
```